pages/vi/profiles/Sravya71.md

```diff
@@ -0,0 +1,10 @@
+####_**Hi, I am Laxmi Sravya Nittala, I am an aspiring Intern for OLE Below are my Details**_:
+1.Name: Laxmi Sravya Nittala
+2.Location\Time Zone: NewJersey\EST
+3.OS Version: Windows 10 Home
+ 
+ >Hi **OLE Team**, I am Sravya, I have done my masters from ![Lamar University][Lamar], which is a state university in Texas.
+ >
+ >I have AWS, CCNA certifications.I am here to learn, Contribute and collaborate with the **OLE Team**. Looking forward to work **OLE**!
+ 
+ [Lamar]:https://www.google.com/search?q=lamar+university&sxsrf=ACYBGNQ54CQIHjUDPGh_Q5SXSsnXNVVNYA:1574975118782&source=lnms&tbm=isch&sa=X&ved=2ahUKEwjTrfDD543mAhXSZd8KHTUGBHEQ_AUoA3oECBEQBQ#imgrc=zYiPE_4VWEHlzM:
```

Comment:
a picture is not working

Reply:
Hi @pattanawadee88 , Attached below is the new raw link:

https://raw.githubusercontent.com/sravya71/Sravya.treehouses.github.io/Branch-sravya-new-branch/pages/vi/profiles/Sravya71.md

Reply:
the link should be included #! before the word 'pages'

Reply:
@pattanawadee88  Below is the requested Link: 
https://raw.githack.com/sravya71/sravya.treehouses.github.io/Branch-sravya-new-branch/#!pages/vi/profiles/Sravya71.md

Thank you,
Sravya.

Reply:
the picture is still not working for me. 
you can check this https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images for reference

Reply:
@pattanawadee88 https://raw.githack.com/sravya71/Sravya.treehouses.github.io/Branch-sravya-new-branch/#!pages/vi/profiles/Sravya71.md  

Can you please check this link??